modules/init-snippet-install-awscli/snippet.tpl

```diff
@@ -1,6 +1,6 @@
 # start snippet - install awscli python cli tools
 ${init_prefix}
-apt-get install -y python-pip
-pip install --upgrade pip==9.0.3
-pip install awscli
+apt update # in case this is missed in the whole user_data. Per my testing, APT won't download data updated within certain amount of time, except security related.
```

Comment:
I'm not strongly objecting, but ideally, we would rely on the operator to do that, so this is not redundant. For example, if the operator is diligent, this module would punish them with extra apt updates than is needed.

Reply:
OK.